content/posts/2023-04-05-microshift-on-crc.md

```diff
@@ -0,0 +1,34 @@
+---
+title: " Introducing MicroShift preset for CRC"
+date: 2023-04-05T07:12:10+05:30
+---
+
+Are you a developer who wants to use OpenShift Kubernetes but doesn't have a powerful system?
```

Comment:
They call it "OpenShift Kubernetes Engine" in the official document which is linked  below. I would replace it everywhere in that post.
I don't know if this should be "Red Hat OpenShift Kubernetes Engine" the first time it's mentioned.